CMakeLists.txt

```diff
@@ -869,10 +869,11 @@ if(USE_OPENSSL_QUIC)
   list(APPEND LIBCURL_PC_REQUIRES_PRIVATE "libnghttp3")
 endif()
 
-if(USE_MBEDTLS OR
-   USE_BEARSSL OR
-   USE_SECTRANSP)
-  message(WARNING "A selected TLS library does not support TLS 1.3.")
+if(USE_BEARSSL)
+  message(WARNING "BearSSL does not support TLS 1.3.")
+endif()
+if(USE_SECTRANSP)
+  message(WARNING "Secure Transport does not support TLS 1.3.")
```

Comment:
What do you think of moving these warnings to the end of their detection blocks? To make them easier to find. It'd also make the warnings appear earlier.

Reply:
Sure, go ahead. The main thing of this PR is that mbedTLS nowadays has TLS 1.3 support.